docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+phpdoc:
+  container_name: phpdoc
+  image: phpdoc:latest
```

Comment:
Since we do not have a registry yet; can you change the `image` reference to a `build` element? That way docker-compose knows how to build/boot the image